lib/wicked/wlan.pm

```diff
@@ -261,6 +262,45 @@ sub hostapd_can_wep {
     return $s !~ m/unknown configuration item 'wep_key0'/i;
 }
 
+sub hostapd_can_key_mgmt {
```

Comment:
```suggestion
sub is_key_mgmt_invalid {
```

Reply:
Changed it to `is_hostapd_supporting_key_mgmt()`